sdk/migrationdiscoverysap/Azure.ResourceManager.MigrationDiscoverySap/tests/Tests/MigrationSapDiscoveryTests.cs

```diff
@@ -0,0 +1,333 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.ResourceManager.MigrationDiscoverySap.Models;
+using Azure.ResourceManager.MigrationDiscoverySap.Tests.Tests.Enums;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Resources;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
```

Comment:
i think record files are missing. have we tried to run this in record mode, and then did it succeed in playback mode?